static/docs/commands-reference/destroy.md

```diff
@@ -43,11 +43,58 @@ $ ls -a
 
 $ dvc destroy
 
-This will destroy all information about your pipelines as well as cache in .dvc/cache.
+This will destroy all information about your pipelines, all data files, as well as cache in .dvc/cache.
 Are you sure you want to continue?
 yes
 
 $ ls -a
 
 .git code.py foo
 ```
+
+#### Cache directory is changed from default
```

Comment:
please check dvc commit, dvc checkout or other commands with multiple examples to make the title rignt